doc/Settings.md

```diff
@@ -105,9 +105,12 @@ The `downloader` setting controls which code is used when downloading packages.
 `wininet` uses the [WinINet](https://docs.microsoft.com/en-us/windows/win32/wininet/about-wininet) APIs, while `do` uses the
 [Delivery Optimization](https://support.microsoft.com/en-us/windows/delivery-optimization-in-windows-10-0656e53c-15f2-90de-a87a-a2172c94cf6d) service.
 
+The `doProgressTimeoutInSeconds` setting updates the number of seconds to wait without progress before fallback. The default number of seconds is 20, minimum is 1 and the maximum is 600. 
```

Comment:
```suggestion
The `doProgressTimeoutInSeconds` setting updates the number of seconds to wait without progress before fallback. The default number of seconds is 60, minimum is 1 and the maximum is 600. 
```

And really the minimum and maximum are not enforced (although minimum should be).

Reply:
Got the numbers from the settings schema: 
https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/settings/settings.schema.0.2.json
Can you update those as well.